pug/page-contents/media_content.html

```diff
@@ -281,14 +281,28 @@ <h4>Initialization</h4>
     var elems = document.querySelectorAll('.slider');
     var instances = M.Slider.init(elems, {
       // specify options here
+      indicatorLabelFunc: (idx, current) => {
+        let label = "Go to slide " + idx;
+        if (current) {
+          label = label + " (Current)";
+        }
+        return label;
+      }
     });
   });
 
   // Or with jQuery
 
-  $(document).ready(function(){
+  $(document).ready(function() {
     $('.slider').slider({
       // specify options here
+      indicatorLabelFunc: (idx, curr) => {
```

Comment:
oh I just saw this, isn't this should be `current` instead of `curr`?

---

btw we might need to comment that `indicatorLabelFunc` is optional, otherwise a newbie might just copy paste it without knowing what it do

Reply:
Fixed this line :)

Btw I've updated the documentation so as to identify this function as optional (directly on its description and in code samples).